gallery/src/components/NxDropdown/NxDropdownPage.tsx

```diff
@@ -198,10 +213,9 @@ const NxDropdownPage = () =>
                 An additional icon-only button which can appear on the right side of a menu row
               </NxTableCell>
               <NxTableCell>
-                Sometimes it is desireable to add a icon-only button to a menu row which performs an action distinct
+                Sometimes it is desirable to add a icon-only button to a menu row which performs an action distinct
                 from clicking on the row itself - for instance, a trashcan button which deletes the row. For this
-                scenario, the icon button, a sibling of the main row button, must include this class and
-                the <code className="nx-code">nx-dropdown-button-container</code> and{' '}
+                scenario, the icon button, a sibling of the main row button, must include this class and{' '}
```

Comment:
Probably worth mentioning that the icon button must appear _before_ the link or the display breaks (because of the right float).

Reply:
> Probably worth mentioning that the icon button must appear before the link or the display breaks (because of the right float).

c2f67692